### DISCUSSION TITLE
user-guide/add-attachments

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#278] [文档评论区]
# user-guide/add-attachments

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/add-attachments

**Comment by jiabayang** [comment]
您好，想请问一下怎么重新自动重新定位附件的位置呢，点击link的文件发现链接不对，应该是在上一级的目录。

**Comment by Maxpicca-Li** [comment]
你好，请问多个附件，zotero如何指定主要附件？比如一篇文章他18年发的有个PDF，19年他在 arxiv上更新了一个PDF，两个PDF都是这个条目的附件，但是双击条目打开的是19年的pdf，我想让他默认打开18年的PDF

**Reply by northword** [reply]
请参阅：https://github.com/orgs/zotero-chinese/discussions/167 。
